internal/cluster/validations/validations.go

```diff
@@ -210,3 +214,10 @@ func ValidateNoProxyFormat(noProxy string) error {
 	}
 	return nil
 }
+
+func ValidateSSHPublicKey(sshPublicKey string) (err error) {
+	if _, _, _, _, err = ssh.ParseAuthorizedKey([]byte(sshPublicKey)); err != nil {
+		err = errors.Errorf("Malformed SSH key: %s", sshPublicKey)
```

Comment:
Nothing useful to print from 'err'?

Reply:
no. the only error it can throw is 'ssh: not found'.